src/atoms/toogle/stories/toogle.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Toggle from '../index';
+import { ToggleProps } from '../types';
+
+export default {
+  title: 'Design System/Atoms/Toggle',
+  component: Toggle,
+  argTypes: {
+    isLabelVisible: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    disabled: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    checked: {
+      control: {
+        type: 'boolean',
+      },
+    },
+  },
```

Comment:
```suggestion
```

You can remove `argTypes` in this case. This is useful when we have multiple stories for the same component, and we want to create default arguments for all.